tests/config/fixtures/config.yaml

```diff
@@ -1,3 +1,5 @@
+# TODO: UPDATE YAML
```

Comment:
What is this?

Reply:
To update the config yaml with the latest changes, meaning that we have to update the tests.

Reply:
I didn't saw that it was the fixtures one, thanks!